### DISCUSSION TITLE
Which country is entirely 1000m above 🌊 level?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#895] [Galaxy Trivia] [answered]
**Bonus:**  What 🦕 is named after the country?

**Comment by peterbaccaro** [comment, marked as answer]
**Lesotho** is a small, mountainous enclave surrounded on all sides by South Africa, and it's the only country in the world that sits entirely above 1,000 meters of elevation.

The dinosaur named after the country Lesotho is called **Lesothosaurus**. Lesothosaurus is an extinct genus of herbivorous dinosaur that lived approximately 200 million years ago during the Early Jurassic period. The name "Lesothosaurus" means "lizard from Lesotho" and was given to the dinosaur because its fossils were discovered in Lesotho, specifically in the Upper Elliot Formation in southern Africa. Lesothosaurus is notable for its small size, estimated to have been about 1 to 1.5 meters in length, and it is considered an early member of the ornithischian group of dinosaurs.
